.gitignore

```diff
@@ -26,7 +26,8 @@ cabal.project.local~
 *.tix
 *.lock
 hie.yaml
+.vscode
```

Comment:
Note that this has nothing to do with the project, whatever concerns your editor is none of our business 😅

You should consider using a global git ignore for your tooling / editor.

Reply:
Btw, I use Arch. ;-)